src/ukify/ukify.py

```diff
@@ -1042,9 +1125,25 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
     ),
 ]
 
-CONFIGFILE_ITEMS = { item.config_key:item
-                     for item in CONFIG_ITEMS
-                     if item.config_key }
+
+CONFIGFILE_CREATE_ITEMS = { item.config_key:item
+                            for item in CONFIG_ITEMS_CREATE
+                            if item.config_key }
+
+
+CONFIGFILE_DISPLAY_ITEMS = { item.config_key:item
+                             for item in CONFIG_ITEMS_DISPLAY
+                             if item.config_key }
+
+
+CONFIGFILE_COMMON_ITEMS = { item.config_key:item
+                            for item in CONFIG_ITEMS_COMMON
+                            if item.config_key }
+
+
+CONFIGFILE_ALL_ITEMS = { **CONFIGFILE_CREATE_ITEMS,
+                         **CONFIGFILE_DISPLAY_ITEMS,
+                         **CONFIGFILE_COMMON_ITEMS }
```

Comment:
Pet peeve, but PEP8 says not to have the additional whitespace after parentheses and braces.
```suggestion
CONFIGFILE_CREATE_ITEMS = {item.config_key:item
                           for item in CONFIG_ITEMS_CREATE
                           if item.config_key}


CONFIGFILE_DISPLAY_ITEMS = {item.config_key:item
                            for item in CONFIG_ITEMS_DISPLAY
                            if item.config_key}


CONFIGFILE_COMMON_ITEMS = {item.config_key:item
                           for item in CONFIG_ITEMS_COMMON
                           if item.config_key}


CONFIGFILE_ALL_ITEMS = {**CONFIGFILE_CREATE_ITEMS,
                        **CONFIGFILE_DISPLAY_ITEMS,
                        **CONFIGFILE_COMMON_ITEMS}
```

Reply:
dumb question: how to run automatic pep8 flake and similar checker for ukify in local? I honestly didn't manage to do that.

Reply:
I'm not sure it would do you all that much good, since we have our own style idiosyncracies and the - I think - most wide-spread autoformatter nowadays (black) is not entirely compatible with PEP8. That being said, `ruff` seems to be the current linter du jour, which I think also incorporates the stuff from `pycodestyle`.